package.json

```diff
@@ -139,12 +138,13 @@
 		"cross-env": "^7.0.3",
 		"depcheck": "^1.4.7",
 		"docdash": "^2.0.2",
-		"eslint": "^8.57.0",
```

Comment:
Isn't the dependency to `eslint` missing now? The ui5-builder PR included `eslint` + `@eslint/js`. I also get both when using `npm init @eslint/config@latest`.

Reply:
Thanks!